deploy/docs/Best_Practices.md

```diff
@@ -784,3 +784,63 @@ metadata:
       - name: KUBERNETES_SERVICE_PORT
         value: '12345'
 ```
+
+## OpenTelemetry queueing and batching
+
+OpenTelemetry goes with several parameters related to queue management.
+
+For [batch processor][batch_processor]:
+
+- `send_batch_size` defines the number of items (logs, metrics, traces) in one batch before it's sent further down the pipeline.
+- `timeout` defines time after which the batch is send regardless of the size (can be lower than `send_batch_size`).
+- `send_batch_max_size` is an upper limit of the batch size.
```

Comment:
when `send_batch_max_size` is in use? It is not clear for me what is the difference between `send_batch_size` and `send_batch_max_size`